src/azul/terraform.py

```diff
@@ -290,6 +290,27 @@ def provider_fragment(region: str) -> JSON:
         return {'provider': f'aws.{region}'}
 
 
+def provision_s3_bucket_public_access_block(tf_config: dict) -> JSON:
```

Comment:
```suggestion
def provision_s3_bucket_public_access_block(tf_config: dict) -> JSON:
```

You're returning the argument, so surely the type of the argument and return value should be the same.